core/src/main/java/com/graphhopper/util/StringUtils.java

```diff
@@ -0,0 +1,117 @@
+/*
+ *  Licensed to GraphHopper GmbH under one or more contributor
+ *  license agreements. See the NOTICE file distributed with this work for 
+ *  additional information regarding copyright ownership.
+ * 
+ *  GraphHopper GmbH licenses this file to you under the Apache License, 
+ *  Version 2.0 (the "License"); you may not use this file except in 
+ *  compliance with the License. You may obtain a copy of the License at
+ * 
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package com.graphhopper.util;
+
+
+/**
+ * This class is a partial Copy of the org.apache.commons.lang3.StringUtils
+ * that can be found here: https://github.com/apache/commons-lang/blob/master/src/main/java/org/apache/commons/lang3/StringUtils.java
+ *
+ * The library can be found here: https://commons.apache.org/proper/commons-lang/
+ *
+ * @author Robin Boldt
+ */
+public class StringUtils {
```

Comment:
Would you mind putting the apache commons files into a separate package (`com.graphhopper.apache(.)commons`?) and use the original license header?